src/likes.js

```diff
@@ -0,0 +1,10 @@
+import { getLikes } from './api';
+
+const dspLikes = async (items) => {
+  const getLike = await getLikes(items[0].id);
+  getLike.forEach((element) => {
+    document.getElementById(`l${element.item_id}`).innerHTML += element.likes || 0;
+  });
+};
+
+export default dspLikes;
```

Comment:
[Optional]
- [ ] Styling of `likes` doesn't look good. You should improve the styling of the likes.

![Screenshot 2021-08-07 at 20-11-09 capstone](https://user-images.githubusercontent.com/1642014/128608399-b51c7ed4-3a89-4a9d-a484-31be2343aea9.png)